smart_ats/jobs/api/views.py

```diff
@@ -42,3 +48,47 @@ def archive(self, request, pk=None, *args, **kwargs):
         job.archive()
         job.save()
         return Response({"message": "Ok"})
+
+
+class JobApplicationViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated, IsApplicantOrJobCompanyAdmin]
+
+    def get_queryset(self):
+        try:
```

Comment:
this does not make sense.
just set the model .
view set has model and lookup key